src/app/core/interfaces/widgets-interface.ts

```diff
@@ -43,10 +43,7 @@ export interface IWidget {
  * @export
  * @interface IPathArray
  */
-export interface IPathArray {
-  /** Key string use to name/identify the path IWidgetPath object. Used for Observable setup */
-  [key: string]: IWidgetPath;
-}
+export type IPathArray = Record<string, IWidgetPath>;
```

Comment:
Can we update and keep the description. It's helpful when you hover the property.

Reply:
Ohh, yeah. Good catch. I'm adding it back in - let me play with it for a second to make sure it shows up properly in hovers.